rendermodules/dataimport/apply_mipmaps_to_render.py

```diff
@@ -74,15 +74,7 @@ def run(self):
         zvalues = self.render.run(
```

Comment:
I think this is a pretty common functionality that we might implement in the StackTransitionModule or a subclass of it.